frontend/src/components/2.Join/JoinGroup.js

```diff
@@ -33,33 +34,33 @@ function GroupCode() {
       <h1 className='Title'>yumble</h1>
       <div className='MakeCentre'>
         <div className='Join_input'>
-          <div className='Username_Box'>
-            Enter Name
-            <div>
-              <UserInput
-                input
-                type='text'
-                inputType='joinGroup' // what is inputType
-                placeholder='e.g Alex'
-                fontSize={3}
-                // on change currently doesnt work
-                onChange={(e) => setName(e.target.value)}
-              ></UserInput>
-            </div>
-          </div>
           <div className='GroupCode_Box'>
-            Enter group code
+            <div className='AddMargin'>Enter group code</div>
             <div>
               <UserInput
                 input
                 type='text'
                 inputType='joinGroup'
-                placeholder='e.g P6aPE'
+                placeholder=' P6aPE'
```

Comment:
Why did you take out the 'e.g.'? I feel like it might help the user?

Reply:
I feel without the e.g looks better and also on many websites I've been to, they don't include that in the placeholder. But I can change it back.

Reply:
Aw nah I think thats alg, I took a look at it properly when I ran it and it looks better actually

Reply:
yea, feel free to change it on your next commit if you feel like it might look better. :)